app-text/zeal/files/add-missing-qt-components.patch

```diff
@@ -0,0 +1,8 @@
+--- a/src/app/CMakeLists.txt
++++ b/src/app/CMakeLists.txt
+@@ -1,4 +1,4 @@
+-find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets REQUIRED)
++find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Concurrent WebEngineWidgets Widgets REQUIRED)

```

Comment:
Is there an upstream ticket for this?

Reply:
Actually this is https://github.com/zealdocs/zeal/commit/c432a0ac22b59ed44bdcec8819c030d993177883

I would recommend importing that as a patch file, for provenance and so the next person to look at this PR doesn't ask you the same question I just did. ;)

Reply:
This is a "release-port" of https://github.com/zealdocs/zeal/pull/1644 (porting required because the release tarball is generated with CRLF line endings and normal git output is rejected because it expects LF).

Reply:
lmao what an absolute joke. Why is the release tarball generated with different line endings!!!

Can you please include a comment at the top of the backport patch referring to where you got it from and that it's a backport? Just add some English prose lines above the `--- a/src/app/CMakeLists.txt` (this is basically what a git-formatted patch is, except with structured comments that git can parse as git metadata).

Reply:
Done. I'd guess they release it on Windows and use git's `autocrlf` so it converts line endings for them on checkout.

Reply:
I am already running zeal-0.7.1 but with QT5, could QT5/6 be a USE flag?

Reply:
> I am already running zeal-0.7.1 but with QT5, could QT5/6 be a USE flag?

In theory it could, but that's not something I'd be ready to contribute now, mostly because of the testing required.

Reply:
+1 for not providing Qt5 option there. For a package without a maintainer, that just leaves more work to do when it is time to drop Qt5WebEngine from all the revdeps for last-rites.